sa/sa.go

```diff
@@ -1565,6 +1565,28 @@ func (ssa *SQLStorageAuthority) GetAuthorizations(ctx context.Context, req *sapb
 	for name, a := range pendingAuthz {
 		authzMap[name] = a
 	}
+
+	// Fetch each of the authorizations' associated challenges
+	for _, authz := range authzMap {
+		var challObjs []challModel
+		_, err = ssa.dbMap.Select(
+			&challObjs,
+			getChallengesQuery,
+			map[string]interface{}{"authID": authz.ID},
+		)
+		if err != nil {
+			return nil, err
+		}
+		var challs []core.Challenge
```

Comment:
nit: since we know the number of challenges this could be simplified to `authz.Challenges = make([]core.Challenge, len(challObjs))` and the assignment simplified to `authz.Challenges[i], err = ...`.

Reply:
Resolved in ccba7fa